src/components/forumComponents/ModalToCreatePost.js

```diff
@@ -1,14 +1,17 @@
 import { Paper } from "@mui/material";
 import TextField from "@mui/material/TextField";
-import Typography from "@mui/material/Typography"
+import Typography from "@mui/material/Typography";
 import { makeStyles } from "@mui/styles";
 import Button from "@mui/material/Button";
-import { useState } from "react";
+import { useState, useEffect, useContext } from "react";
 import { collection, addDoc } from "firebase/firestore";
 import { db } from "../../index";
 import CloseIcon from "@mui/icons-material/Close";
+import { UserContext } from "../../controllers/UserContext";
 
-export const ModalToCreatePost = ({ userData, setStateOfModal }) => {
+export const ModalToCreatePost = ({ setStateOfModal }) => {
+  const { uid, email, username, avatarUrl, setAvatarUrl } =
+    useContext(UserContext);
   const useStyles = makeStyles({
```

Comment:
dobrze by było przenieść wszystkie style do oddzielnych plików bo to trochę zaciemnia obraz